app/controllers/professors_controller.rb

```diff
@@ -0,0 +1,52 @@
+# Module for Professor endpoint
+
+module Sinatra
+  module UMDIO
+    module Routing
+      module Professors
+
+      	# make sure database 'umdprof' exists!!
+        def self.registered(app)
+
+          app.before '/v0/professors*' do
```

Comment:
makes sense to generalize this logic rather than just copying from `courses_controller`